module13and14/LoopCanvas/LoopCanvas/App/LoopCanvasApp.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  LoopCanvasApp.swift
+//  LoopCanvas
+//
+//  Created by Peter Rice on 5/30/24.
+//
+
+import SwiftUI
+
+@main
+struct LoopCanvasApp: App {
+  var body: some Scene {
+    WindowGroup {
+      ContentView()
```

Comment:
consider removing ContentView from the mix since all it does it go to Main view.

Reply:
Good suggestion!